app/components/province-organization-select.hbs

```diff
@@ -1,16 +1,17 @@
 <div class={{if @error "ember-power-select--error"}}>
   <PowerSelect
     @disabled={{if @disabled true false}}
-    @allowClear={{true}}
+    @allowClear={{not @selectedMunicipality}}
```

Comment:
Same as for the admin unit component, I think the component should not handle this but the caller should instead